lib/import-export-visitor.js

```diff
@@ -415,17 +414,11 @@ function getBlockBodyInfo(visitor, path) {
     // If parent is a Program, we may need to add a "use strict"
     // directive to enable const/let in Node 4.
     needToAddUseStrictDirective = visitor.enforceStrictMode;
-
     break;
 
   case "BlockStatement":
     body = parent.body;
-
-    if (hasOwn.call(parent, "start")) {
```

Comment:
The `BlockStatement`s we create in the `default` case below may not have a `.start` property, so this code was trying to be defensive about that.